apis/stardust/types/index.d.ts

```diff
@@ -695,3 +702,7 @@ declare namespace stardust {
 
 }
 
+export interface Menu {
+    addItem: (input: { translation: string; icon: string; tid: string; select: () => Promise<void> }) => void;
+}
+

```

Comment:
Not sure where this come from?

Reply:
Ah this one, in the above lines (160) I set menu type as `Menu`, it has certain properties, to set their types I chose to add an interface for it.

Reply:
Oh..! Ok I'll remove it then!

Reply:
BTW @T-Wizard , as I see, the file has been modified manually several times by others! Is it something wrong in my case ?

Reply:
you can (and if relevant should) regenerate the file by running `yarn spec` but you should not do any extra edit of file after that as they are lost next time the file is generated